src/flows_dst.rs

```diff
@@ -599,6 +608,69 @@ impl<'a> ProcessElderChange<'a> {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct ProcessMerge<'a>(pub &'a mut MemberState);
+
+impl<'a> ProcessMerge<'a> {
+    pub fn start_event_loop(&mut self) {
+        self.set_is_active(true);
+        self.0
+            .action
+            .send_rpc(Rpc::Merge(self.0.action.our_section()));
```

Comment:
See `send_node_approval_rpc`, I think I've tried to create named functions in Action for these RPCs. It seemed to make sense to me. Not 100% sure that it the right thing, but it would be consistent.

Reply:
Should `send_rpc` be made private then?

Reply:
Maybe, not sure it is the right way to go.